crates/arroyo-df/src/extension/updating_aggregate.rs

```diff
@@ -147,6 +148,17 @@ impl ArroyoExtension for UpdatingAggregateExtension {
             physical_plan_type: Some(PhysicalPlanType::Aggregate(Box::new(combine_aggregate))),
         };
 
+        let flush_interval = std::env::var(UPDATE_AGGREGATE_FLUSH_MILLIS_ENV)
+            .ok()
+            .map(|s| {
+                let Ok(millis): Result<u64, _> = s.parse() else {
+                    return plan_err!("Failed to parse {} to a number", s);
```

Comment:
This should still probably reference what the number is (i.e., that it's the UPDATE_AGGREGATE_FLUSH_MILLIS_ENV value)  otherwise it lacks context for the user.